src/replays/_replayswidget.py

```diff
@@ -273,6 +274,11 @@ def myTreeDoubleClicked(self, item):
             replay(item.filename)
 
     def updatemyTree(self):
+        modification_time = os.path.getmtime(util.REPLAY_DIR)
+        if self.myTree.modification_time < modification_time:  # anything changed?
```

Comment:
Just a style thing, I'd reverse branch order (>= instead) - check if we should bail out first. Lets us put the modification_time setting line outside of condition branches, too.

Reply:
yes, better. (but is == then)